main.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"github.com/kirikami/go_exercise_api/config"
+	"github.com/kirikami/go_exercise_api/database"
+	mw "github.com/kirikami/go_exercise_api/middleware"
+	"github.com/kirikami/go_exercise_api/routes"
+	"github.com/labstack/echo"
+	"github.com/labstack/echo/engine/standard"
+	"github.com/labstack/echo/middleware"
+
+	_ "github.com/uber-common/zap"
+)
+
+var (
+	conf     *config.Configuration
+	DBConfig config.DatabaseConfig
+	port     string
+)
+
+func init() {
+	configfile := flag.String("config", "config.json", "Config for connection to database")
+	conf = config.MustNewConfig(*configfile)
+	DBConfig = conf.DatabaseConfig
+	port = fmt.Sprintf(":%d", conf.ListenAddress)
+}
+
+func main() {
+	db := database.MustNewDatabase(DBConfig)
+
+	server := echo.New()
+
+	server.Use(middleware.Recover())
+	server.Use(middleware.Logger())
+	server.Use(mw.UseConfig(db, conf))
+
+	aut := server.Group("/auth")
```

Comment:
@Kirikami `echo` has endpoint `groups`, use that feature and add versioning to endpoints, endpoint URL should contain version as well